src/install/frontend.py

```diff
@@ -120,6 +122,18 @@ def _install_docker_images(radare):
         raise InstallationError(f'Failed to pull pdf report container:\n{docker_process.stdout}')
 
 
+def _copy_mime_icons():
```

Comment:
Could we maybe symlink the directory rather than copying it?

Reply:
I don't think that this would be a good idea (for safety reasons) even if it works (I'm not that proficient with path traversal attacks though so I can't say for sure)